src/CovidTracker/plot_time_series.py

```diff
@@ -1,11 +1,31 @@
-def plot_ts(df):
+from get_covid_data import get_covid_data 
+import matplotlib.pyplot as plt
```

Comment:
is matplotlib used in your function anywhere?

Reply:
lol no, i turned out deciding to use altair, good catch!